Tests/AppTests/AllTests.swift

```diff
@@ -0,0 +1,121 @@
+// Copyright Dave Verwer, Sven A. Schmidt, and other contributors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Testing
+import Dependencies
+
+
+@Suite(.dependency(\.date.now, .t0)) struct AllTests { }
```

Comment:
In order to be able to bring up a number of test dbs at the start of the app tests and tear them down afterwards, all the tests need to belong to a single suite. The easiest way to do this is to nest all existing tests under a new top level suite.

This suite will then later gain the trait

```diff
- @Suite(.dependency(\.date.now, .t0)) struct AllTests { }
+ @Suite(.setupDatabasePool, .dependency(\.date.now, .t0)) struct AllTests { }
```

NB: `AllTests` isn't an ideal name I've just realised, because we have a couple of other test suites that aren't nested. I'll rename this to `AllAppTests` after all is done. It'd be too much rebasing for little gain at this point.